packages/web-components/src/popover/popover.ts

```diff
@@ -0,0 +1,129 @@
+import { attr, FASTElement, Updates } from '@microsoft/fast-element';
+import { arrow, computePosition, offset } from '@floating-ui/dom';
+
+export class Popover extends FASTElement {
+  @attr({ mode: 'boolean' })
+  public open: boolean = false;
+
+  @attr({ mode: 'boolean' })
+  public arrow: boolean = false;
+
+  @attr
+  public position: 'top' | 'bottom' | 'left' | 'right' = 'top';
+
+  public popoverContentRef: FASTElement; // @FIXME: what type?
+  public arrowRef: FASTElement;
+  public anchorRef: HTMLElement[];
+
+  public connectedCallback() {
+    super.connectedCallback();
+    this.openChanged(); // hack
+  }
+
+  public disconnectedCallback() {
+    super.disconnectedCallback();
+    this.hidePopover();
+  }
+
+  protected openChanged() {
```

Comment:
From a code quality standpoint, I think it'll make the most sense to keep these associated with the corresponding `attr` and `observables` - what do you think? Those are "tightly" coupled basically because each attr/observable gets a corresponding changed function like this. We find that it's easier to grok by having those right next to one another.